src/methods.cpp

```diff
@@ -3299,9 +3299,9 @@ PyObject* meth_get_phy_firmware_version(PyObject* self, PyObject* args)
 {
 	PyObject* obj = NULL;
 	char phy_indx = 0;
-	unsigned int phyVersion = NULL;
-	unsigned int functionError = 1;
-	if (!PyArg_ParseTuple(args, arg_parse("Oi|ii:", __FUNCTION__), &obj, &phy_indx, &phyVersion, &functionError))
+	int phy_version = NULL;
+	int function_error = 1;
```

Comment:
`phy_version` and `function_error` shouldn't be passed into `PyArg_ParseTuple`. Python doesn't send these as references/pointers.